include/blmc_robots/n_joint_blmc_robot_driver.hpp

```diff
@@ -350,7 +350,7 @@ class NJointBlmcRobotDriver
         for (size_t i = 0; i < motor_boards.size(); i++)
         {
             motor_boards[i] = std::make_shared<blmc_drivers::CanBusMotorBoard>(
-                can_buses[i], 1000, 10);
+                can_buses[i], 1000, 50);
```

Comment:
I just used some value that is significantly higher than the old one to see if it works.  I can try to reduce it a bit and see if it is still okay.

There is a quite problematic issue with our interface related to this: If the motor board goes to fault state (e.g. because of this timeout), it disables the motor but the higher level interface does not detect this.  This means that the corresponding motor will be disabled without the user being aware of it.  We need to think of a good way to report this failure to the user in a way that it cannot be ignored (maybe by throwing an exception?).

Reply:
I reduced it to 20ms. I started the robot 50 times in a loop without the timeout triggering.

Reply:
yes, actually i think throwing an exception would be fine, since this should never happen. but we have to make sure we don't do something which would be a problem for the locomotion guys.

Reply:
Timing issue was because initialisation was not running in a real-time thread.  Fixed in #35.  With the fix the increased timeout is not needed, so I dropped the corresponding commits from this PR.